.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+
+*E.g. when doing ..., ... unexpectedly happened*
```

Comment:
This is what I'm not a fan of in the PR template. This adds noise to the template and the author. Instead, can we put all the examples and helpful test in markdown comments, so the author doesn't have to always delete things?